frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -0,0 +1,171 @@
+# Automated Annotation of Single Cell Data (EXPERIMENTAL)
+
+CZ CELLxGENE Annotate enables label transfer and reference mapping from expertly curated reference atlases to your newly generated and QC'd single cell datasets. Your data can be annotated using CELLxGENE Desktop along with any of our available label transfer and reference mapping models. When you run a CELLxGENE annotation model, your dataset will contain the following new information:
```

Comment:
This sentence is good if the audience is a computational biologist. If you're targeting an experimental biologist you may want to lead with a sentence that uses simpler language. Then, if it's necessary, explain that it achieves prediction through label transfer and reference mapping.